src/ui/common/src/components/integrations/cards/lambdaCard.tsx

```diff
@@ -0,0 +1,21 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+import { Integration, LambdaConfig } from '../../../utils/integrations';
+
+type Props = {
+  integration: Integration;
+};
+
+export const LambdaCard: React.FC<Props> = ({ integration }) => {
```

Comment:
What's the difference between lambdaCard and lambdaDetailCard? The contents seem similar.

Reply:
Hi Eunice, the only difference between "lambdaCard" and "lambdaDetailCard" is that we start a new line for each entry. For example, the regular card has "name: name", the detail card has "name: /n name". My initial figma design to have have a new line for each title and content but now that I have thought about it for the sake for simplicity maybe we should stick with the same card as before?